### PR TITLE
Doc flowchart class diagram added to static and README

### PR DESCRIPTION
**Pull request by Gui-FernandesBR**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`python setup.py install`) was run locally and any changes were pushed
- [ ] Lint (`black rocketpy`) has passed locally and any fixes were made for failures
- [ ] All tests (`pytest --runslow`) have passed locally

## Pull request type
 
 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

RocketPy's code did not have any unit tests setup.

## What is the new behavior?

- I am adding an image to the static folder in this repository
- This image will be refferenced on the README by the following path: "https://github.com/Projeto-Jupiter/RocketPy/tree/master/docs/static/"

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by Gui-FernandesBR** [comment]
Dear reviewers, 
Please take into consideration the following website/path, which is used on the README.md file, will not be availlable until the PR is merged: `https://github.com/Projeto-Jupiter/RocketPy/tree/master/docs/static/Fluxogram-Page-2.svg`

**Comment by PatrickSampaioUSP** [comment]
Could you please try to use Relative Links? I think this will ensure that the image will be available and will be more logical when someone is reading the readme source (and set the standard for future image addition in the readme)

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files

**Comment by Gui-FernandesBR** [comment]
@PatrickSampaioUSP , thanks for your solid review.
All done as requested.

File: README.md
Comment (giovaniceotto) [inline README.md:173]: I like what you have done very much! Creating a new section for the Class diagram is a good option. Did you also consider adding the diagram after line 183? There, RocketPy's main classes are mentioned for the first time, perhaps helping to contextualizing the whole thing.

File: README.md
Comment (giovaniceotto) [inline README.md:172]: I like what you have done very much! Creating a new section for the Class diagram is a good option. Did you also consider adding the diagram after line 183? There, RocketPy's main classes are mentioned for the first time, perhaps helping to contextualizing the whole thing.

Comment (Gui-FernandesBR) [reply README.md:172]: Thanks @giovaniceotto ,

No, I did not consider it. Acctually I didn't saw this part, let me arrange thw whole thing again

**Review by giovaniceotto — APPROVED** [review]
Looks perfect now. Thanks for your contribution!

**Comment by giovaniceotto** [comment]
Any updates here? Are we ready to merge?

**Comment by Gui-FernandesBR** [comment]
I was waiting for @AdrianoAntongiovanni 's review. Probably this week he will take a look at it.

Is there any urgency? If so please feel free to merge

**Comment by giovaniceotto** [comment]
No urgency!

**Pull request merged**
